src/utils/work/fetchWork.js

```diff
@@ -1,14 +1,14 @@
 /* eslint-disable no-underscore-dangle */
-import hash from 'object-hash';
+import { MD5 } from 'object-hash';
```

Comment:
Feel free to keep it like this if you think it's out of scope, but we need to change this hashing algorithm to something more secure. Right now it is not a problem, but in the future if we decide to use this hash for something else, it can become a bit problem.

Reply:
If we're going to change the algorithm, I think it should be part of another ticket